src/lava/magma/core/model/py/ports.py

```diff
@@ -26,36 +26,40 @@ class AbstractPyPort(AbstractPortImplementation):
     @abstractmethod
     def csp_ports(self) -> ty.List[AbstractCspPort]:
         """
-        Abstract property to get all csp_ports of a PyPort.
+        Abstract property to get all connected PyPorts (csp_ports) of a PyPort.
```

Comment:
Does not seem right. csp_ports are the fundamental CSP ports that implement the CSP protocol. PyPorts are not the csp_ports.